packages/squiggle-lang/src/rescript/FR/FR_List.res

```diff
@@ -75,6 +79,26 @@ module Internals = {
 }
 
 let library = [
+  Function.make(
+    ~name="length",
+    ~nameSpace,
+    ~output=EvtNumber,
+    ~requiresNamespace=false,
```

Comment:
Most of the other functions require a namespace, and `List.length` probably should too.

Reply:
I see. I copied the template from another function which had requiresNamespace set to false. What does this property stand for? For example this can be seen on reduce